### ISSUE TITLE
False positive adblock agent detection

### ISSUE THREAD

**Issue by scripthunter7** [AdguardTeam/AGLint#104]
In adblock agent detection we need to handle the following case:

```
[$adg-modifier]##[class^="adg-"]
```

Fast check makes it look like agent, since it starts with `[` and ends with `]`.

This can be easily avoided by a cosmetic rule separator search.
